libevmasm/Assembly.h

```diff
@@ -216,6 +246,7 @@ class Assembly
 	std::string m_name;
 
 	langutil::SourceLocation m_currentSourceLocation;
+	std::vector<std::shared_ptr<std::string const>> m_sources;
```

Comment:
I originally expected this to contain some source code. `sourceList` would be better. It still suggests that it's the source code but at least it matches the `sourceList` key in the input JSON so it might still be intuitive enough if you know the structure.
```suggestion
	std::vector<std::shared_ptr<std::string const>> m_sourceList;
```

But ideally, this would be something even more descriptive. E.g. `m_sourceUnitNames`.